KarhooUISDK/Screens/BookingRequestScreen/FormBookingRequestScreen/FormBookingRequestPresenter.swift

```diff
@@ -94,6 +94,19 @@ final class FormBookingRequestPresenter: BookingRequestPresenter {
     func isKarhooUser() -> Bool {
         return karhooUser
     }
+    
+    func addPassengerDetails() {
```

Comment:
Can we add unit tests for the logic?

Reply:
there are 2 more flows to be covered, I added my last comment in the PR that tests should be written for all of them. I am not sure the logic I’ve added in this ticket is final, that’s why I have only added the protocol conformance for now.

Reply:
@mohadian I have added a test as you requested :) it is only for the guest flow as of now, because the other flows are not yet implemented, they will be implemented in one of the tickets from this sprint.